canonical-web-and-design/jaas-dashboard
# Add status icon to owners grouping tables

## Done

- Add status icon to owners grouping tables
- Forced all credential values to be lowercase
- Add snapshot testing for owner/status tables

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Group by 'owner'
- Verify that values in the status column now have an appropriate status icon

## Details

Fixes #261 

## Screenshots

<img width="393" alt="Screenshot 2020-01-07 at 14 41 01" src="https://user-images.githubusercontent.com/505570/71903346-c7acd880-315b-11ea-9d1c-08a83401530c.png">
